src/main/java/lotto/domain/LottoTicketMachine.java

```diff
@@ -0,0 +1,25 @@
+package lotto.domain;
+
+import static lotto.constants.Condition.MAXIMUM_VALUE;
+import static lotto.constants.Condition.MINIMUM_VALUE;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.List;
+import lotto.vo.request.PurchaseAmountRequest;
+
+public class LottoTicketMachine {
+    static public IssuedLottos generateLottos(PurchaseAmountRequest purchaseAmount) {
+        PurchaseNumber purchaseNumber = purchaseAmount.calculatePurchaseNumber();
+
+        Integer count = 0;
+        List<Lotto> lottoTickets = purchaseNumber.createLottoTickets();
+        while(purchaseNumber.lessThanPurchaseNumber(count)) {
+            List<Integer> numbers = Randoms.pickUniqueNumbersInRange(MINIMUM_VALUE, MAXIMUM_VALUE, 6);
+            Lotto lotto = new Lotto(numbers);
+            lottoTickets.add(lotto);
+            count += 1;
+        }
+
+        return new IssuedLottos(lottoTickets);
```

Comment:
(이런 방법도 있어요!)
```suggestion
             return new IssuedLottos(IntStream.iterate(0, purchaseNumber::lessThanPurchaseNumber, i -> i + 1)
                .mapToObj((ignore) -> Randoms.pickUniqueNumbersInRange(MINIMUM_VALUE, MAXIMUM_VALUE, 6))
                .map(Lotto::new)
                .toList());
```

IntStream의 iterate를 활용해서 반복문 + 조건식의 형태를 갖출 수 있어요!